infra/indexers/src/utils.ts

```diff
@@ -0,0 +1,20 @@
+// Fonction pour décoder une chaîne hexadécimale en une chaîne de caractères
```

Comment:
Should the comments be in english? 😂

Reply:
Also I believe starknet.js has a function to convert short strings to/from felts. it's in the `shortString` namespace.

Reply:
Sorry about the french comments :D

Ok for the use of the starknet.js function